LoginFormInHtmlAndCss-Penyez/style.css

```diff
@@ -0,0 +1,118 @@
+
+    {
+    margin: 0;
+    padding: 0;
+        box-sizing: border-box;
+        font-family: "Poppins", sans-serif;
+    }
+
+    body{
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        min-height: 100vh;
+        background: url('gradient-particle-wave-background_23-2150517309.jpeg') no-repeat;
+        background-size: cover;
+        background-position: center;
+    }
+
+    .wrapper {
+    width:420px;
+    backdrop-filter:blur(20px);
+    border: 2px solid rgba(255, 255, 255, .2);
+        box-shadow: 0 0 10px rgba(0,0,0, .2);
```

Comment:
Buradaki 3 rakam ne anlama geliyor. box-shadow: 0 0 10px;

Reply:
Birinci ve ikinci değer sırasıyla yatay ve dikey ofset diye geçiyor. Bunlar gölgenin yatay ve dikey düzlemdeki konumunu belirtiyor. Üçüncü değer ise gölgenin ne kadar bulanık olacağını ifade ediyor. rgba da senin de bildiğin üzere renk ve saydamlığı belirliyor.